AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/MutationStateStructures.dfy

```diff
@@ -462,6 +470,14 @@ module {:options "/functionSyntax:4" } MutationStateStructures {
          Types.KeyStoreAdminException(
            message:="Terminal Properities contain illegal Encryption Context! There are some resereved Encryption Context Keys!"));
 
+    var originalHV := MutablePropertiesJsonToHierarchyVersion(OriginalJson);
+    var terminalHV := MutablePropertiesJsonToHierarchyVersion(TerminalJson);
```

Comment:
Blocking: Use these values on line 487 and 492.

Reply:
Resolved in https://github.com/aws/aws-cryptographic-material-providers-library/pull/1458/commits/d7ac72e9148ad8ebaaa55c0f12c6f26328aebc80, but I got carried away with other things as well